src/controllers/authors.js

```diff
@@ -0,0 +1,17 @@
+import { User } from '../models';
+import { responseFormat, errorResponseFormat } from '../utils';
+
+const getAuthors = async (req, res) => {
+  try {
+    const authors = User.findAll({
+      where: { role: 'author' },
+    });
+
+    if (authors) {
+      return res.status(200).json(responseFormat({ status: 'success', message: authors }));
+    }
+  } catch (error) {
+    return res.status(500).json(errorResponseFormat({ status: 'error', message: error }));
```

Comment:
Error value is not a message. Inspect the error and construct the appropriate message